### DISCUSSION TITLE
Getting a team reference with Get-PnPTeamsTeam

### DISCUSSION THREAD

**Discussion by Ofer-Gal** [pnp/powershell#156] [Q&A] [answered]
I am not sure if connecting with a URL and certificate is the right way to get a Team
$appConn = Connect-PnPOnline  -Url $appUrl -ClientId $ClientId -CertificatePath $certPath -CertificatePassword $CertPass  -Tenant $tenant -ReturnConnection
Then 
$sourceTeam = Get-PnPTeamsTeam -Identity "aebabbef-180f-49f6-92c9-be4201be286d"  -Connection $appConn
gets me a null $sourceTeam  (Not "Team not found")
What am I doing wrong.

I am hoping to make a tool to copy / move Teams Channel contents from one team to another within same tenant.
Is getting channels then setting on another Team will do it ?

Thank you all

**Comment by erwinvanhunen** [comment, marked as answer]
There was actually an issue with the cmdlet. Please check out the version we release the coming night, available tomorrow: version 1.2.5-nightly.
